rust/src/datatypes.rs

```diff
@@ -14,6 +14,7 @@
 // KIND, either express or implied.  See the License for the
 // specific language governing permissions and limitations
 // under the License.
+use std::fmt;
```

Comment:
nit: I think, based on other source files, we keep a new line after the license header.